framework/src/base/FEProblem.C

```diff
@@ -3691,6 +3691,9 @@ FEProblem::adaptMesh()
       computeMarkers();
     if (_adaptivity.adaptMesh())
       meshChanged();
+
+    // Show adaptivity progress
+    _console << std::flush;
```

Comment:
Agreed. You don't really notice unless you are actively watching it. It's
very obvious if you use many Adaptivity cycles with a high `max_h_level`
such that a lot of time is spent doing the adaptivity. Without the flush,
the console will can appear to hang for several seconds (problem dependent
of course) before proceeding with the next time step, while with the flush,
you see the output from each cycle appear one-by-one. Just take any test
problem that uses multiple `cycles_per_step` and set `max_h_level = 0` so
eventually you get lots of nodes and the adaptivity steps take longer and
longer.

On Wed, May 4, 2016 at 10:36 AM, Cody Permann <notifications@github.com>
wrote:

> In framework/src/base/FEProblem.C
> <https://github.com/idaholab/moose/pull/6896#discussion_r62049874>:
>
> > @@ -3691,6 +3691,9 @@ FEProblem::adaptMesh()
> >        computeMarkers();
> >      if (_adaptivity.adaptMesh())
> >        meshChanged();
> > +
> > +    // Show adaptivity progress
> > +    _console << std::flush;
>
> @jessecarterMOOSE <https://github.com/jessecarterMOOSE> - This is tricky
> to test. Does it appear to be working for you?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/6896/files/938afedf29fb7e30041f61c75fa7815afb84469c#r62049874>
>



-- 
Jesse Carter, Ph.D.

cell: 979-229-6266
work: 412-476-6727
jesse.carter@gmail.com